README.md

```diff
@@ -1,14 +1,44 @@
+### **Installation**
+Detail documentation on how to setup Chatbot NER on your system using docker is available [here](docs/install.md). We
+are working on building a pip package for the same.
 
-2. [CSV file structure and adding/removing entities to DataStore](docs/adding_entities.md): *Document on how to create your own entity and store it to DataStore*
+### **Supported Entities**
 
-3. [Approach used](docs/approach.md): *Document on the architecture along with different detection logics (entity types) and built-in entities*
+| Entity type   | Code reference       | Description                              | example                           | Supported languages - **ISO 639-1** code |
+| :------------ | -------------------- | :--------------------------------------- | --------------------------------- | ---------------------------------------- |
+| Time          | [TimeDetector](https://github.com/hellohaptik/chatbot_ner/tree/develop/ner_v2/detectors/temporal/time) | Detect time from given text. | tomorrow morning at 5, कल सुबह ५ बजे, kal subah 5 baje | 'en', 'hi', 'gu', 'bn', 'mr', 'ta' |
+| Date          | [DateAdvancedDetector](https://github.com/hellohaptik/chatbot_ner/tree/develop/ner_v2/detectors/temporal/date) | Detect date from given text | next monday, agle somvar, अगले सोमवार | 'en', 'hi', 'gu', 'bn', 'mr', 'ta' |
+| Number        | [TimeDetector](https://github.com/hellohaptik/chatbot_ner/tree/develop/ner_v2/detectors/numeral/number]) | Detect number and respective units in given text | 50 rs per person, ५ किलो चावल, मुझे एक लीटर ऑइल चाहिए | 'en', 'hi', 'gu', 'bn', 'mr', 'ta' |
+| Phone number  | [PhoneDetector](https://github.com/hellohaptik/chatbot_ner/tree/develop/ner_v2/detectors/pattern/phone_number) | Detect phone number in given text | 9833530536, +91 9833530536, ९८३३४३०५३५ | 'en', 'hi', 'gu', 'bn', 'mr', 'ta' |
+| Email         | [EmailDetector](https://github.com/hellohaptik/chatbot_ner/tree/develop/ner_v1/detectors/pattern/email) | Detect email in text | hello@haptik.co | 'en' |
+| Text          | [TextDetector](https://github.com/hellohaptik/chatbot_ner/tree/develop/ner_v1/detectors/textual/text) | Detects custom entities in text string using full text search in Datastore or based on contextual model| Order me a **pizza**, **मुंबई** में मौसम कैसा है   | Search supported for 'en', 'hi', 'gu', 'bn', 'mr', 'ta', Contextual model supported for 'en' only|
+| PNR           | [PNRDetector](https://github.com/hellohaptik/chatbot_ner/tree/develop/ner_v1/detectors/pattern/pnr) | Detects PNR (serial) codes in given text. | My flight PNR is 4SGX3E | 'en' |
 
-4. [API calls](docs/api_call.md): *A brief document on the api calls for different detection logics along with its necessary output*
+There are other custom detectors such as [city](https://github.com/hellohaptik/chatbot_ner/tree/develop/ner_v1/detectors/textual/city),
+[person name](https://github.com/hellohaptik/chatbot_ner/tree/develop/ner_v1/detectors/textual/name),
```

Comment:
city and person name should only be mentioned if they are useable out of the box, which they are not without the variants data

Reply:
Budget can be mentioned instead

Reply:
Done. City data get's indexed on startup. Removed person name and added budget.